src/oidbridge/CMakeLists.txt

```diff
@@ -21,11 +21,93 @@
 # FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
 # IN THE SOFTWARE.
 
-cmake_minimum_required(VERSION 3.10.0)
+cmake_minimum_required(VERSION 3.24.0)
```

Comment:
This is a bit tricky... I assumed the Ubuntu GHA runners came with software with the same default versions as what you get from `apt install`, but it seems that this is not the case. In short, Ubuntu 20.04 which is the reference OS and version that OID aims to support comes with `CMake 3.16.3`. Ubuntu 22.04 comes with `CMake 3.22.1`, so both workflows should have failed. I will have to change CI workflows to make sure the Ubuntu runners use the default sw versions.

Which feature from this PR needs 3.24.0?

Reply:
It is for `VERSION_EQUAL` check

Reply:
> both workflows should have failed

Interesting that they succeeded.

Reply:
I've reverted this version bump, since `VERSION_EQUAL` appears in [docs for CMake 3.0.2](https://cmake.org/cmake/help/v3.0/command/if.html).

Reply:
Sorry, my bad. 
Actually this version bump is for `PATH_EQUAL`, that has appeared in [3.24](https://cmake.org/cmake/help/latest/release/3.24.html)

Reply:
> > both workflows should have failed
> 
> Interesting that they succeeded.

Found out why: the GHA runner has CMake version 3.28.3 installed - https://github.com/actions/runner-images/blob/ubuntu20/20240212.2/images/ubuntu/Ubuntu2004-Readme.md#tools. Don't know if it's possible to bypass it, otherwise I will move the build step to docker containers from these files https://github.com/OpenImageDebugger/dockerfiles

Reply:
I suggest to use modern CMake. `PATH_EQUAL` is short, concise and convenient. 

OpenImageDebugger is a tool for software developers. Anyway they would install something from non-standard repositories.

Why not to suggest CMake update?